geolocator_platform_interface/lib/src/geolocator_platform_interface.dart

```diff
@@ -159,6 +159,21 @@ abstract class GeolocatorPlatform extends PlatformInterface {
     throw UnimplementedError('getPositionStream() has not been implemented.');
   }
 
+  /// Asks the user for Temporary Precise location access (iOS 14 or above).
+  ///
+  /// Throws a [AccuracyDictionaryNotFoundException] when the key
+  /// `NSLocationTemporaryUsageDescriptionDictionary` has not been set in the
+  /// `Infop.list`.
+  /// Throws a [PreciseAccuracyEnabledException] when the user already gave
+  /// permission to use Precise Accuracy location fetching.
+  /// Throws a [ApproximateLocationNotSupportedException] when Approximate
+  /// Location is not supported (iOS 13 or below).
```

Comment:
This should also include the other platforms (Android, Web, potential future platforms). Maybe something like:

```suggestion
  /// Throws a [ApproximateLocationNotSupportedException] when Approximate
  /// Location is not supported (iOS 13 or below and on other platforms).
```